dockerfiles/hpcc/templates/localroxie.yaml

```diff
@@ -31,7 +31,7 @@ spec:
 {{ include "hpcc.addConfigVolume" . | indent 6 }}
 {{ include "hpcc.addVolumes" $ | indent 6 }}
 ---
-{{- include "hpcc.generateConfigMap" (dict "root" $ "me" $roxie ) -}}
+{{- include "hpcc.generateConfigMap" (dict "root" $ "component" "roxie" "me" $roxie ) -}}
```

Comment:
There is scope to create this dict just once and pass to a few places - not sure if doing so makes things clearer or less clear